package.json

```diff
@@ -40,6 +40,7 @@
     "babel-preset-react": "~6.11.1",
     "browser-sync": "~2.15.0",
     "browser-sync-webpack-plugin": "~1.1.2",
+    "bs-fullscreen-message": "~1.1.0",
```

Comment:
From reading the docs of webpack-hot-middleware, this seems different. You still need webpack-hot-middleware to be put along your entries and somewhere along with browsersync. This is the setup we have on react-instantsearch